lib/bloc/leaves/load_apply_leave_bloc.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:async';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:saasify/data/models/leaves/load_apply_leave_model.dart';
+import 'package:saasify/di/app_module.dart';
+import 'package:saasify/repositories/leaves/leaves_repository.dart';
+
+import '../../caches/cache.dart';
+import 'load_apply_leave_event.dart';
+import 'load_apply_leave_state.dart';
+
+class LoadApplyLeaveBloc extends Bloc<LoadApplyLeaveEvent, LoadApplyLeaveState> {
```

Comment:
Name should be LeaveBloc instead of LoadApplyLeaveBloc

Reply:
done